regparser/layer/terms.py

```diff
@@ -3,7 +3,12 @@
 from itertools import chain
 import re
 
-from inflection import pluralize
+import inflection 
+try:
+    del inflection.PLURALS[inflection.PLURALS.index(('(?i)(p)erson$', '\\1eople'))]
+except ValueError:
+    pass
```

Comment:
Roger. It might be worthwhile to add a wrapper that took the term and returned (potentially more than one) plural + singular forms. Future work, mayhaps.